tools/tools_test.go

```diff
@@ -393,6 +469,33 @@ func TestParser(t *testing.T) {
 			tmpl:    list,
 			calls:   nil,
 		},
+		{
+			name: "list trailing ]",
+			inputs: []string{
+				"[",
+				"{",
+				"\"name\": \"get_conditions\", ",
```

Comment:
i've often seen`[{` and `{"` come down together would be nice to have the parsing test that pattern as well so we're making less assumptions

Reply:
Updated one of the tests to use the single "token" `[{`